apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Tests/Logging/GoogleLoggerTest.cs

```diff
@@ -127,25 +131,74 @@ public void Log_EmptyFormattedMessage()
         [Fact]
         public void Log()
         {
+            var message = "a {things} message with stuff";
+            var logStr = "log";
+
             var labels = new Dictionary<string, string> { { "some-key", "some-value" } };
             Predicate<IEnumerable<LogEntry>> matcher = logEntries =>
             {
                 LogEntry entry = logEntries.Single();
                 KeyValuePair<string, string> label = entry.Labels.Single();
-                return entry.LogName == new LogName(_projectId, _logName).ToString() &&
+                var json = entry.JsonPayload.Fields;
+                return entry.LogName == new LogName(_projectId, _baseLogName).ToString() &&
                     entry.Severity == LogLevel.Error.ToLogSeverity() &&
                     string.IsNullOrWhiteSpace(entry.Trace) &&
                     entry.Timestamp.Equals(Timestamp.FromDateTime(s_dateTime)) &&
-                    entry.TextPayload == Formatter(_logMessage, s_exception) &&
+                    json["message"].StringValue == "a log message with stuff" &&
```

Comment:
Is the raw, non-formatted log message (e.g. `"a {things} message with stuff"`) also logged somewhere? It would be nice if we can filter on the message format regardless of the substituted values.

Reply:
Yes it is, added an explicit check for this.

Reply:
Awesome!